activestorage/lib/active_storage/service/disk_service.rb

```diff
@@ -9,7 +9,7 @@ module ActiveStorage
   # Wraps a local disk path as an Active Storage service. See ActiveStorage::Service for the generic API
   # documentation that applies to all services.
   class Service::DiskService < Service
-    attr_reader :root
+    attr_accessor :root
```

Comment:
I think this should also be a separate commit to the doc updates.

Reply:
https://github.com/rails/rails/pull/42163

Reply:
I think this can be reverted if we're handling it in another PR.

Reply:
Yep! I was gonna do that after https://github.com/rails/rails/pull/42163 was merged, since technically the code samples in this PR won't work without this code change.

Reply:
But that's overly pedantic, I'll just do it now.